content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -729,6 +730,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `ServiceAppProtocol`: Enables the `AppProtocol` field on Services and Endpoints.
 - `ServiceLBNodePortControl`: Enables the `spec.allocateLoadBalancerNodePorts`
   field on Services.
+- `ServiceLoadBalancerClass`: Enables the `LoadBalancerClass` field on Services. See [Service Type=LoadBalancer Class Field](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/1959-service-lb-class-field) for more details.
```

Comment:
The Service concept is one of the most important abstraction in Kubernetes. Any feature changes to Service warrants a paragraph or two in the user facing documentation, I believe. Pointing users to the KEP may not be a good idea because a KEP contains too much details irrelevant to them.

Reply:
I see, I am willing to add a document for this one, but I am super busy for this week :(, and I saw @reylejano  said docs review ddl  in the day after tmr.

I wonder could we just keep this now, since this field is in the alpha stage and default disabled, and I add a detailed document for this in a follow-up PR and merge it in the next release?
@tengqm

Reply:
@XudongliuHarold I'm fine with it. At least it is better than having no pointers, :)

Reply:
@XudongLiuHarold ultimately that question (do we accept a feature that is alpha and with incomplete documentation) is mainly one for SIG Release.

SIG Docs does prefer documentation aimed at end users, even for alpha features. We want readers to be confident they can understand what they might gain by enabling the feature, and potentially also what risks this might imply.

@reylejano policy in this area has changed for v1.21, right? What's the latest thinking?

Reply:
@XudongLiuHarold  I've brought up this PR to the 1.21 release lead, 1.21 emeritus lead, sig network ([via slack](https://kubernetes.slack.com/archives/C09QYUH5W/p1616735284120000)) and  cross-posted to sig docs ([via slack](https://kubernetes.slack.com/archives/C1J0BPD2M/p1616735337051900)).  I've asked if another contributor can help complete the documentation or if this should be pushed to 1.22. I'll comment if there are any replies or if a decision has been made.

Reply:
Thanks @reylejano , I can work on this next week, just wonder if it can catch up deadline for 1.21?

Reply:
@XudongLiuHarold Thank you for working on this over the weekend. This is updating this PR with the [slack thread ](https://kubernetes.slack.com/archives/C09QYUH5W/p1616779968130400?thread_ts=1616735284.120000&cid=C09QYUH5W),  @XudongLiuHarold will target documentation completion by eod Sunday and reviewed by sig-network colleagues by Tuesday

Reply:
Detail documentation added, thx.